frontend/src/components/DebugInformation/DebugInformation.tsx

```diff
@@ -56,13 +54,46 @@ const copyToClipboardOptions = {
   message: 'We failed to automatically copy the text, please use: #{key}, Enter\t'
 };
 
+// Will be shown in Kiali Config and hidden in Additional state
+const propsToShow = ["accesibleNamespaces", "authStrategy", "clusters", "gatewayAPIEnabled",
```

Comment:
```suggestion
const propsToShow = ["accessibleNamespaces", "authStrategy", "clusters", "gatewayAPIEnabled",
```

It looks like this is not a typo but just matching a mistake in the JSON.  To avoid confusion let's fix things such that all occurrences of "accesibleNamespaces" are changed to"accessibleNamespaces".  I'll make suggested changes where I see them, in this review...

Reply:
Yes, better to fix the typo